games/a.yaml

```diff
@@ -477,7 +478,7 @@
 
 - name: AstroMenace
   meta:
-    genre: [shmup]
+    genre: [Shmup]
```

Comment:
Wikipedia indicates that the correct capitalization for the abbreviation is `shmup` (all lowercase) https://en.wikipedia.org/wiki/Shoot_'em_up

I'm not in disfavor of keeping it as Shmup to match the rest of the capitalized tags but it does look pretty weird with a capital S. Would be nice to have a third opinion on this.